ocdskingfisherarchive/config.py

```diff
@@ -0,0 +1,89 @@
+import configparser
```

Comment:
Why do we have this? I don't think the reqs require connecting to a DB.

Reply:
1) There will be much more config than just a DB by the end, tons of directory paths, S3 creds, ...

2) "Get all collections that are finished and are not transforms" from the document is currently by an SQL query and that's in the google doc - that's how the old system was looking up that info. We could look up Scrapy logs and parse them for final messages I suppose, but this seems easier.